core/include/kernel/semihosting.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024 Andes Technology Corporation
+ */
+#ifndef __KERNEL_SEMIHOSTING_H
+#define __KERNEL_SEMIHOSTING_H
+
+#include <sys/fcntl.h>
+#include <sys/types.h>
```

Comment:
_sys/types.h_ is not strictly needed, only _stdint.h_ for `uintptr_t` and _stddef.h_ for `size_t`.

Reply:
Changed in https://github.com/OP-TEE/optee_os/pull/6700/commits/37c6df05cd56ba3e16b9c88e011d0517bc89d49c